app/controllers/workshops_controller.rb

```diff
@@ -10,6 +10,7 @@ def new
   def create
     @workshop = Workshop.new(workshop_params)
     @workshop.user = current_user
+    raise
```

Comment:
there is still a raise here, i think you forgot to take it out :)

Reply:
My bad, I removed it.